### PR TITLE
Add "check-code" for local publisher-proxy development

### PR DESCRIPTION
**Pull request by radufa**
**Description**

Changes proposed in this pull request:

- Add a new target "code-check" which uses "check-imports", "fmt-local", "vet-local", "lint"

**Related issue(s)**

### CODE REVIEW

File: components/event-publisher-proxy/Makefile [hunk 1]
```diff
@@ -26,8 +26,11 @@ dep-status-local:
 mod-verify-local:
 	GO111MODULE=on go mod verify
 
+# Use it before commit
+check-code: check-imports fmt-local vet-local lint
+
 test-local:
-	GO111MODULE=on go test ./...
+	GO111MODULE=on go test ./... -coverprofile cover.out
```
Comment (k15r) [inline components/event-publisher-proxy/Makefile:33]: do we really need to always write the coverage to a file? Usually developers use an IDE to check the the coverage and these then also take care of generating the coverage for you

**Review by k15r — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

File: components/event-publisher-proxy/Makefile (hunk 1 above)
Comment (radufa) [reply components/event-publisher-proxy/Makefile:33]: Added only to have the same coverage behavior as in "eventing-controller"

**Pull request merged**